com.unity.cinemachine/Runtime/Upgrader/CinemachineUpgraderDriver.cs

```diff
@@ -0,0 +1,25 @@
+#if UNITY_EDITOR
+using UnityEngine;
+
+namespace Cinemachine.Upgrader
+{
+    [ExecuteInEditMode]
+    public class CinemachineUpgraderDriver : MonoBehaviour
```

Comment:
How do you intend this to be used?  Is it just for internal development?

Reply:
It is only for internal development :)